### DISCUSSION TITLE
[jotai-devtools] missing `redux` peer dependency

### DISCUSSION THREAD

**Discussion by rtritto** [pmndrs/jotai#1721] [General]
## Summary
`redux` is a peer dependency of `@redux-devtools/extension` (dependency of `jotai-devtools`).
It's a transitive peer dependency and it should be added in _package.json_ of `jotai-devtools`.

Alternatives:
- copy the used code of `@redux-devtools/extension` and remove it
- move `@redux-devtools/extension` to dev dependencies and add it during build

## Step to reproduce
1. `yarn init -y`
2. `yarn set version canary`
3. `yarn add jotai jotai-devtools react`
4. output:
```console
➤ YN0000: ┌ Resolution step
➤ YN0002: │ jotai-devtools@npm:0.1.0 [16463] doesn't provide redux (pae9f5), requested by @redux-devtools/extension
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed
➤ YN0000: Done with warnings
```

**Comment by dai-shi** [comment]
Or just use `any`.
Related PR: https://github.com/pmndrs/jotai/pull/1049

What I like would be to use the correct type while developing (= dev dependency) and replace the type with `any`.

**Comment by dai-shi** [comment]
Moved to discussions. I hoped to transfer the issue to the jotai-devtools repo, but it's not possible. Let me create a new issue.

👉  https://github.com/jotaijs/jotai-devtools/issues/25
